<ui version="4.0" >
 <class>SmilesDialog</class>
 <widget class="QDialog" name="SmilesDialog" >
  <property name="windowModality" >
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>191</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Import Smiles</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QRadioButton" name="fileRadioButton" >
       <property name="text" >
        <string>File:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="fileLineEdit" />
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="browseButton" >
       <property name="text" >
        <string>Browse</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QRadioButton" name="smilesRadioButton" >
       <property name="text" >
        <string>Smiles:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="smilesLineEdit" />
     </item>
     <item row="2" column="0" >
      <widget class="QRadioButton" name="databaseRadioButton" >
       <property name="text" >
        <string>Database:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="databaseLineEdit" />
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>ID Code (3 letters):</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <item>
        <widget class="QLineEdit" name="idLineEdit" >
         <property name="text" >
          <string>LIG</string>
         </property>
         <property name="maxLength" >
          <number>3</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>234</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons" >
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SmilesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel" >
     <x>157</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SmilesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>322</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel" >
     <x>286</x>
     <y>190</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>browseButton</sender>
   <signal>clicked()</signal>
   <receiver>fileRadioButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>356</x>
     <y>28</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
